### PR TITLE
corrected some grammar and images in readme and commented out tests in lab

### PR DESCRIPTION
**Pull request by MarcosPalaciosIII**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,28 +32,28 @@ When we are coding, we have to be sure that our code is working as we expect. Mo
 
 We need to execute our tests. First of all, fork and clone this repo into your Github account. Once you are done, open the file `SpecRunner.html` with your browser.
 
-In the beginning, you will see all the tests in yellow because the tests we have to implement are commented. (except for one, that througs an error, no worries :wink:)
+In the beginning, you will see all the tests in yellow because the tests we have to implement are commented. (except for one, that throws an error, no worries :wink:)
 
-![image](https://user-images.githubusercontent.com/23629340/36485165-c51a20fc-171b-11e8-854c-a83fb7dec036.png)
+![image](http://materials.ironhack.com/s/HkAOp3cvQ)
```
Comment (nizaroni) [inline README.md]: This is the URL of note but you want the URL of the image file. You can find it inside the markdown code of the editor. 
screen shot 2018-09-06 at 17 33 55

File: README.md [hunk 2]
```diff
@@ -32,28 +32,28 @@ When we are coding, we have to be sure that our code is working as we expect. Mo
...
   expect(varX).toBe(true);
-}); 
+});
 ```
 
-When we uncomment the line and refresh de `SpecRunner.html` page, we will see something like that:
+When we uncomment the line and refresh the `SpecRunner.html` page, we will see something like that:
 
-![image](https://user-images.githubusercontent.com/23629340/36485153-bcc140ca-171b-11e8-9dac-5c94be14bb68.png)
+![image](http://materials.ironhack.com/s/rJoopn9PQ)
```
Comment (nizaroni) [inline README.md]: Same as above. Change to the image URL.

**Comment by MarcosPalaciosIII** [comment]
corrected the image path and pushed changes

**Comment by nizaroni** [comment]
We are all set! Thanks Marcos. These changes will be included in the next curriculum deployment.

**Pull request merged**